### PR TITLE
Adjust license information to reflect Perl license.

### PR DESCRIPTION
**Pull request by bk2204**
There were several conflicting license statements in the files: some
indicated the Perl license, some indicated the cPanel variant of the BSD
license, and some indicated a proprietary cPanel license.  After
clarification from cPanel's legal department, update all the licenses to
refer to the Perl license.

### CODE REVIEW

**Comment by bk2204** [comment]
I have an S/MIME-signed statement from @escherlat approving this in his official capacity, if that should be necessary.

File: lib/cPanel/TQSerializer/Storable.pm [hunk 1]
```diff
@@ -1,10 +1,5 @@
 package cPanel::TQSerializer::Storable;
 
-# cpanel - cPanel/TQSerializer/Storable.pm        Copyright(c) 2014 cPanel, Inc.
-#                                                           All rights Reserved.
-# copyright@cpanel.net                                         http://cpanel.net
-# This code is subject to the cPanel license. Unauthorized copying is prohibited
-
 use Storable();
 
 #use warnings;
```
Comment (ColMelvin) [inline lib/cPanel/TQSerializer/Storable.pm]: I see no mention of the Perl license in this file.

File: lib/cPanel/TQSerializer/YAML.pm [hunk 2]
```diff
@@ -1,10 +1,5 @@
 package cPanel::TQSerializer::YAML;
 
-# cpanel - cPanel/TQSerializer/YAML.pm            Copyright(c) 2014 cPanel, Inc.
-#                                                           All rights Reserved.
-# copyright@cpanel.net                                         http://cpanel.net
-# This code is subject to the cPanel license. Unauthorized copying is prohibited
-
 use YAML::Syck ();
 
 # with no bless flags
```
Comment (ColMelvin) [inline lib/cPanel/TQSerializer/YAML.pm]: I see no mention of the Perl license in this file. Is having it in the separate POD file sufficient?

Comment (bk2204) [inline lib/cPanel/TQSerializer/YAML.pm]: It probably is, but just to be sure, I'll fix it.

**Comment by bk2204** [comment]
Updated.

**Pull request merged**